players/utils.py

```diff
@@ -14,7 +21,7 @@ def add_player_to_team_and_group(team, player):
     team.pending_player.clear()
 
 
-def last_and_first_picking_persons(team):
+def last_and_first_picking_persons(team: Team) -> tuple[Profile, Profile]:
```

Comment:
Jeżeli tu używasz `tuple` (z małej litery) a nie `Tuple` (wielką literą `from typing import Tuple`) to znaczy, że używasz pythona 3.9+. W takim razie nie musisz importować też `from typing import List` tylko możesz używać wbudowanego `list` (małą literą).

Importować `List` czy `Tuple` (wielkimi literami) trzeba tylko przy pythonie 3.8 i niższym.